src/main/java/jpa/domain/LineRepository.java

```diff
@@ -13,4 +14,6 @@
 
     Line findByColor(String color);
 
+    @Query("select l from Line l join fetch l.lineStations where l.name = :name")
```

Comment:
쿼리를 한 줄로 쓰면 읽기도 힘들고 유지보수하기 어려워집니다

```suggestion
    @Query("select l " +
            "from Line l " +
            "join fetch l.lineStations " +
            "where l.name = :name ")
```